internal/core/command/init.go

```diff
@@ -33,7 +33,12 @@ import (
 )
 
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the command service.
-func BootstrapHandler(wg *sync.WaitGroup, ctx context.Context, startupTimer startup.Timer, dic *di.Container) bool {
```

Comment:
The function signature already fits on a single line (within the 120-word limit), should you wish to collapse the lines.